ci-operator/jobs/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main-periodics.yaml

```diff
@@ -1,4 +1,71 @@
 periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 00 13 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: openshift-kni
+    repo: eco-ci-cd
+  labels:
+    ci-operator.openshift.io/cluster: build05
+    ci-operator.openshift.io/variant: cnf-core-4.16
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-kni-eco-ci-cd-main-cnf-core-4.16-eco-ci-cd
+  reporter_config:
```

Comment:
Please add your job name in the .config.prowgen in the config/eco-ci-cd folder, and remove from here (this way is soon to be deprecated).

Reply:
added to .config.prowgen. But there is no way to remove it from here, it was aded by make update

Reply:
Excellent, I added a comment on this topic.